plugins/com.google.cloud.tools.eclipse.appengine.deploy.ui/plugin.xml

```diff
@@ -99,4 +89,14 @@
       </adapt>
     </definition>
   </extension>
+
+  <extension point="org.eclipse.ui.console.consolePageParticipants">
+    <consolePageParticipant
+        class="com.google.cloud.tools.eclipse.appengine.deploy.ui.DeployConsolePageParticipant"
+        id="com.google.cloud.tools.eclipse.appengine.deploy.consolePageParticipant">
+      <enablement>
+        <instanceof value="com.google.cloud.tools.eclipse.appengine.deploy.ui.DeployConsole"/>
```

Comment:
This can't use the same `org.eclipse.ui.console.consoleTypeTest` property test as above?